engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlan.java

```diff
@@ -307,6 +307,11 @@ public void sendMetrics(Collection<String> metricNames)
       throw new UnsupportedOperationException("Not supported yet.");
     }
 
+    @Override
+    public OperatorMeta getOperator()
```

Comment:
Looks like there is a method already in the class that is returning this information?